.github/action/integration-tests-mage-os/action.yml

```diff
@@ -0,0 +1,100 @@
+name: "Integration Tests Mage-OS in Warden"
+author: "Vladyslav Podorozhnyi"
+description: "A Github Action that run Integration Tests of Mage-OS in warden."
+
+## TODO: use inputs as list of tests to be executed
+
+inputs:
+  search:
+    required: true
+    default: "elasticsearch:7.15.1"
+    description: "The search engine to use."
+
+  rabbitmq:
+    required: true
```

Comment:
can this be optional? (I believe you can run mageos without rabbit)

Reply:
Hi 😀

sure it can be - all services can be enabled / disabled by using these input params.

have a look on workflows - I’m using your matrix with for determining should we use rabbit or no, redis (with specific version ) or not use it at all. Same for the search and version and distributive of database .